### ISSUE TITLE
Full card description not read out in NVDA

### ISSUE THREAD

**Issue by ad9242** [mi6/ic-ui-kit#1486]
## Summary of the bug 
When using a screen reader with IcCard component, the full card description is not read out

## 🪜 How to reproduce 
1) Go to page the [component gallery page](https://design.sis.gov.uk/components).
2) Use the NVDA screen reader
3) Navigate to one of the gallery cards using either the virtual cursor or the tab key.
4) See that only the link is announced and not the card description.

## 🧐 Expected behaviour 
The full card description should be read out in NVDA, as it is in VoiceOver

## 📝 Acceptance Criteria  
_Given_ that a card in the component gallery is navigated to with NVDA SR,
_When_ it is navigated to using either virtual cursor or tab focus,
_Then_ its contents should be announced in full, including the description.

## Additional info 
This ticket replaces problem 2 mentioned in https://github.com/mi6/ic-design-system/issues/718

**Comment by GCHQ-Developer-299** [comment]
This issue is replicable with any card that has `clickable=true` 
So I can test [Clickable Card stories](https://mi6.github.io/ic-ui-kit/branches/develop/web-components/?path=/story/web-components-card--clickable-link)

**Comment by GCHQ-Developer-299** [comment]
Unable to replicate this issue today, using an AWS VM with NVDA. Will contact a teammate who has NVDA on their machine to see if they can replicate it:
![Image](https://github.com/mi6/ic-ui-kit/assets/111292477/0d0f6386-7dd5-4dff-9b5d-57dcb602dc7c)
